install.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/sh
-
+printf '\t\e[1;34m%-6s\e[m\n' "Add your user to video group if you haven't already"
```

Comment:
Can you remove this statement, please?
I won't know or remember what colour this gives. :)

Use normal `echo` instead.